source/plugin/configuration/defaults.rst

```diff
@@ -0,0 +1,49 @@
+=========================
+Packaging Default Configs
+=========================
+
+Example: Loading a default config from the plugin jar file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: java
+
+    import java.net.URL;
```

Comment:
Any reason for excluding the imports for `ConfigurationLoader`, `CommentedConfigurationNode`, and `HoconConfigurationLoader`? This is the first code block for this page, or would they be deemed unnecessary?